### ISSUE TITLE
Multiple applied themes?

### ISSUE THREAD

**Issue by bobrocke** [osxpeppermint/peppermint#87]
The Dev Tools inspector shows I have two themes active at the same time:

```
<div id="editor" class="editor ace_editor ace_dark ace-Color It Bob ace-Bob_Colors" style="font-size: 12px; font-family: SourceCodePro-Regular;">
```

Those are `ace-Color It Bob` and `ace-Bob_Colors`.

And `ace-Color It Bob` doesn't look like it should even be legal.

What's going on here?

**Comment by osxpeppermint** [comment]
Hmm... That's a good question and a good catch too. I'm having a look into it.

**Comment by osxpeppermint** [comment]
Issue fixed as of the upcoming 1.5 beta 20.

Related with: https://github.com/osxpeppermint/peppermint/issues/88
